src/net.h

```diff
@@ -132,7 +132,7 @@ class BanMan
     // between nodes running old code and nodes running
     // new code.
     ~BanMan();
-    BanMan(fs::path ban_file, CClientUIInterface* client_interface);
+    BanMan(fs::path ban_file, CClientUIInterface* client_interface, int64_t default_ban_time);
```

Comment:
I cant say I really understand the desire to avoid the argparsing dep, when you include <util.h> either way. Seems like useless indirection.

Reply:
So that we can test independent of the cmdline args.